tools/generate-operator-patches.ts

```diff
@@ -57,19 +58,26 @@ const AliasMethodOverrides = {
 };
 
 function generateNewOperatorFileContents (op:OperatorWrapper): OperatorWrapper {
+  var header = `/**
```

Comment:
can these variable can be `const`? I know it's script and not for actual codebases though.